README.md

```diff
@@ -1 +1,20 @@
-# syncer
\ No newline at end of file
+# syncer
+
+Syncs galaxy integration forks with upstream
+
+## Example Usage (workflow)
+```yaml
+name: sync-upstream
+on:
+  schedule:
+    - cron: '30 5,17 * * *'
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GOG-Nebula/syncer@v0
+        with:
+          upstream: owner/repo
+          reviewers: username1,username2
```

Comment:
You will want to update the reviewers.

Reply:
> so that we would have to use artificial github-account like FriendsOfGalaxyBot. But probably don't have to put its secret to each repo - only to this github action. Probably. Maybe. I don't know...

Yeah just seeing that, its pretty unfortunate not sure why thats a thing. 

> Alternatively we can just list all repo members with other step and pass the resutl to create PR step.

I think this is a great workaround. We could even automatically remove the upstream owner from the list.